### DISCUSSION TITLE
Switching to 8x Dec/Jan Timeframe -what's different in Catalog INIT JSON Files?

### DISCUSSION THREAD

**Discussion by tech-maven-geospatial** [TerriaJS/terriajs#4892] [Q&A]
I saw that you have a converter to change the schema of the Catalog JSON files
What's different about the new catalog format?

We've developed a Catalog Generator /Builder/Editor Tool (let's you configure catalog export as JSON)
https://mapsdata.world/earthexplorer_cataloggenerator/
Most likely we will opensource this tool. to give back to the FOSS4G community.
We are using this in Earth Explorer 3D Map Mobile App (iOS and Android) and Windows Desktop App http://earthexplorer.world
https://apps.apple.com/np/app/earth-explorer-3d-map-app/id1485111764
https://play.google.com/store/apps/details?id=com.app.earthexplorer&hl=en_US

**Comment by rowanwins** [comment]
It's hard to list all the changes in a succinct way, alot stays the same but there are some common things that people will run into (eg `groups` no longer have `items`, they have `members`).

I'd suggest checking out our [catalog-converter repo](https://github.com/TerriaJS/catalog-converter) and [app](https://github.com/TerriaJS/catalog-converter-ui). As we convert many of our apps over we're updating those repos and so hopefully when it comes time for others to transition they'll be able to make good use of these.
